zookeeper-server/src/test/java/org/apache/zookeeper/test/ClientTest.java

```diff
@@ -140,15 +140,15 @@ public void testACLs() throws Exception {
                 zk.create("/acltest", new byte[0], testACL, CreateMode.PERSISTENT);
                 fail("Should have received an invalid acl error");
             } catch (InvalidACLException e) {
-                LOG.info("Test successful, invalid acl received : " + e.getMessage());
+                LOG.debug("Test successful, invalid acl received : " + e);
             }
             try {
                 ArrayList<ACL> testACL = new ArrayList<ACL>();
                 testACL.add(new ACL(Perms.ALL | Perms.ADMIN, new Id()));
                 zk.create("/nullidtest", new byte[0], testACL, CreateMode.PERSISTENT);
                 fail("Should have received an invalid acl error");
             } catch (InvalidACLException e) {
-                LOG.info("Test successful, invalid acl received : " + e.getMessage());
+                LOG.debug("Test successful, invalid acl received : " + e);
```

Comment:
Thanks for the catch! After a closer look it is an intended exception so that

```java
LOG.info("Test successful, invalid acl received : {}", e.getMessage());
```

reflect the intention. Updated.